bigflow/__init__.py

```diff
@@ -1,19 +1,24 @@
-from . import bigquery
 from . import resources
-from . import monitoring
 from .workflow import Workflow, Definition
 from .configuration import Config
-from .build import default_project_setup
+
 
 __all__ = [
     # core
     'Workflow',
     'Definition',
     'Config',
-    'default_project_setup',
```

Comment:
why?

Reply:
It is not needed, and also it results in importing build module and trying to access xmlrunner